go/oasis-test-runner/oasis/keymanager.go

```diff
@@ -247,9 +299,18 @@ func (net *Network) NewKeymanager(cfg *KeymanagerCfg) (*Keymanager, error) {
 		return nil, errors.Wrap(err, "oasis/keymanager: failed to create keymanager subdir")
 	}
 
+	// HACK HACK HACK: Not sure how to fit this into the fixture stuff.
```

Comment:
What's the problem? The only major difference to a `KeyManagerPolicyCfg` is that the `Runtime` would be referenced by index (so it can be serialized) instead of being a pointer to a runtime object.

I would change this slightly so that the network would have a list of key manager policies, similar to how it has a list of runtimes. Then at network setup time, you would iterate over them and set them up. The fixture would then just initialize that list, similar to how it does with runtimes.